client/src/pages/GamePage/filedsData.js

```diff
@@ -0,0 +1,227 @@
+export const fieldsData = [
```

Comment:
Zamiast tak na sztywno to bym wygenerował za pomocą `Array(15).fill(Array(15).fill(1).map((_, idx) => idx))` i tylko te specjalne zmienił na coś innego albo coś podobnego

Reply:
Albo jeszcze zamiast wypełniać od 0 do 14 to bym zostawił puste po prostu